src/app/containers/WebVitals/index.jsx

```diff
@@ -10,15 +10,20 @@ import { RequestContext } from '#contexts/RequestContext';
 const WebVitals = () => {
   const { personalisationEnabled } = useContext(UserContext);
   const { pageType } = useContext(RequestContext);
-  const { enabled } = useToggle('webVitalsMonitoring');
+  const { enabled, value: toggleSampleRate } = useToggle('webVitalsMonitoring');
 
   // Checks if readers have opted into performance tracking and if the feature toggle is enabled
   const isWebVitalsEnabled = personalisationEnabled && enabled;
 
+  const sampleRate = parseInt(
+    toggleSampleRate || process.env.SIMORGH_WEBVITALS_DEFAULT_SAMPLING_RATE,
+    10,
+  );
```

Comment:
If `parseInt(value, base)` is being used here purely to convert to the number type, we could use `Number(value)` instead.